src/SCREENS.ts

```diff
@@ -714,6 +715,10 @@ const SCREENS = {
         TRANSACTION_VIOLATION_CREATE: 'Debug_Transaction_Violation_Create',
         TRANSACTION_VIOLATION: 'Debug_Transaction_Violation',
     },
+    SCHEDULE_CALL: {
+        BOOK_CALL: 'ScheduleCall_Book_Call',
```

Comment:
I think we should update to. Call is redundant 

```suggestion
        BOOK: 'ScheduleCall_Book',
```